src/docs/tutorial-transaction-process/customize-pricing/index.md

```diff
@@ -654,3 +654,25 @@ included in the price on the booking page. In addition, the Flex Console
 transaction price breakdown also shows the cleaning fee.
 
 ![Cleaning fee in booking breakdown in Flex Console](./consoleBookingBreakdown.png)
+
+<extrainfo title="Add cleaning fee to email notifications">
+If you want to add the cleaning fee to your email notifications, you will need to add it to the email templates.
```

Comment:
```suggestion
If you want to add the cleaning fee to your email notifications, you will need to update the email template: money-paid-html.html.
```

Not absolutely necessary, but as this is a tutorial, I think it should be as unambiguous as possible.

Reply:
Good point - there are several templates where the change needs to be made, so I'll list them here!